contents/blog/company-survey-h1-2024.md

```diff
@@ -0,0 +1,99 @@
+---
+date: 2024-06-25
+title: What employees really think about PostHog
```

Comment:
Semantics maybe, but we always say 'team member' rather than 'employee'

Reply:
i like semantics but in this instance are semantics getting in the way of a good headline? 'team members' doesn't have the same ring as 'employees' for me